packages/manager/src/features/Images/EditImageDrawer.tsx

```diff
@@ -0,0 +1,161 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { APIError, Image, UpdateImagePayload } from '@linode/api-v4';
+import { updateImageSchema } from '@linode/validation';
+import * as React from 'react';
+import { Controller, useForm } from 'react-hook-form';
+
+import { ActionsPanel } from 'src/components/ActionsPanel/ActionsPanel';
+import { Drawer } from 'src/components/Drawer';
+import { Notice } from 'src/components/Notice/Notice';
+import { TagsInput } from 'src/components/TagsInput/TagsInput';
+import { TextField } from 'src/components/TextField';
+import { useUpdateImageMutation } from 'src/queries/images';
+
+import { useImageAndLinodeGrantCheck } from './utils';
+
+interface Props {
+  image?: Image;
+  onClose: () => void;
+  open: boolean;
+}
+export const EditImageDrawer = (props: Props) => {
+  const { image, onClose, open } = props;
+
+  const { canCreateImage } = useImageAndLinodeGrantCheck();
+
+  const defaultValues = {
+    description: image?.description ?? undefined,
+    label: image?.label,
+    tags: image?.tags,
+  };
+
+  const {
+    control,
+    formState,
+    handleSubmit,
+    reset,
+    setError,
+  } = useForm<UpdateImagePayload>({
+    defaultValues,
+    mode: 'onBlur',
+    resolver: yupResolver(updateImageSchema),
+    values: defaultValues,
+  });
+
+  const { mutateAsync: updateImage } = useUpdateImageMutation();
+
+  React.useEffect(() => {
+    if (open) {
+      reset();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [open]);
+
+  const onSubmit = handleSubmit((values) => {
+    if (!image) {
+      return;
+    }
+
+    updateImage({ imageId: image.id, ...values })
+      .then(onClose)
+      .catch((errors: APIError[]) => {
+        for (const error of errors) {
+          if (
+            (error.field && error.field === 'label') ||
```

Comment:
Does this one condition need to be different?

```suggestion
            error.field === 'label' ||
```